src/components/db/QueryPageTopStack.tsx

```diff
@@ -39,13 +26,10 @@ export const QueryPageTopStack = ({ matches = 0 }: { matches?: number }) => {
   const isSearchRoute = router.route.startsWith("/search");
 
   const { fileName, sourceLanguage } = useDbQueryParams();
-  const dbView = useAtomValue(currentViewAtom);
+  const dbView = useAtomValue(currentDbViewAtom);
 
-  const { isSettingsOpen, setIsSettingsOpen } = useSettingsDrawer();
-
-  const [open, setOpen] = React.useState(false);
-  const handleOpen = () => setOpen(true);
-  const handleClose = () => setOpen(false);
+  const setIsSourceTreerOpen = useSetAtom(isDbSourceBrowserDrawerOpenAtom);
```

Comment:
typo

Reply:
....noooooo "treeer" is a thing! :flushed: :facepalm: